.github/workflows/linters.yml

```diff
@@ -42,7 +42,257 @@ jobs:
           node-version: "12.x"
       - name: Setup Stylelint
         run: |
-          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          
+html,
+body {
+  margin: 0;
+  padding: 0;
+  padding-bottom: 2em;
+
+  --shadow: 0 2px 10px -1px rgba(0, 0, 0, 0.8);
+  --shadow-accent: 0 2px 50px -1px rgba(0, 0, 0, 0.8);
+  --color-primary: #227452;
+}
+
+body {
+  display: block;
+  background: #fff;
```

Comment:
- [ ] The main container of your project should have a `display: grid`. Kindly, wrap your project's section with a main container.
```
<body>
    <div class="main-container>
        <section class="">
             .....
        </section>

          <section class="">
               .....
          </section>
    </div>
</body>
```
And then, give the main container a `display: grid;`

Reply:
thanks for your kind review but this is linters file. do I need to change it also?

Reply:
Hello dear, not in the linters file, make the change in the HTML file. Sorry for that inline comment on the wrong file.
Best,

Reply:
Grids are already applied according to the template layout. Why do I need to add display:grid?

Reply:
This will help you once you'll start working on the desktop version. But, as you said , I thought that you displayed it as block.
So, kindly submit your PR for re-review as you have already displayed with ~~display: grid~~

Reply:
The display: grid; is already added in CSS of .works. Kindly check line 253 of the CSS file I had already included the display: grid;.

Reply:
![asdf](https://user-images.githubusercontent.com/77225127/159096915-caa88a68-7e9d-4f07-8d5b-588faab237b4.PNG)
You can see from this image the first reviewer didn't check the CSS file thoroughly